stubs/praw/models/redditors.pyi

```diff
@@ -0,0 +1,12 @@
+from typing import Generator
+
+from .base import PRAWBase as PRAWBase
+from .reddit.redditor import Redditor
+from .reddit.subreddit import Subreddit
+
+
+class Redditors(PRAWBase):
+    def new(self, **generator_kwargs: str) -> Generator[Subreddit]: ...
```

Comment:
The docstring said they were descendent of Subreddit, but they should be Redditor from common sense. However, as the docstring said Subreddit, I made them return Subreddits.